app/src/main/java/ar/com/wolox/android/training/ui/errors/ErrorHandler.java

```diff
@@ -9,8 +9,14 @@ public static String getErrorMessage(ErrorCode errorCode) {
             case INVALID_EMAIL:
                 return "El email ingresado es incorrecto";
```

Comment:
I think that you changed this in other branch. Just  don't forget it! 😛

Reply:
Yes! Here: https://github.com/wolox-training/dm-android/blob/home-page-3/app/src/main/java/ar/com/wolox/android/training/ui/errors/ErrorHandler.java